config/toml_test.go

```diff
@@ -49,9 +49,143 @@ func TestEnsureTestRoot(t *testing.T) {
 	// make sure config is set properly
 	data, err := ioutil.ReadFile(filepath.Join(rootDir, "config.toml"))
 	require.Nil(err)
-	assert.Equal([]byte(testConfig("anonymous")), data)
+	assert.Equal([]byte(defaultConfigHardcoded), data)
 
 	// TODO: make sure the cfg returned and testconfig are the same!
 
 	ensureFiles(t, rootDir, "data", "genesis.json", "priv_validator.json")
 }
+
+const defaultConfigHardcoded = `# This is a TOML config file.
```

Comment:
does it mean we'll have to change this every time we change the config?

Reply:
yea.

Reply:
can't we generate it on the fly or something?

Reply:
it's only used by the test. The actual config is generated on the fly, but then the test would be redundant, no? Could use a `strings.Contains` instead to check that key elements of the config are present; something like:
```
var elems = []string{"moniker", "seeds", "proxy_app", "fast_sync", "create_empty_blocks"}
for _, e := range elems {
        if !strings.Contains(configFile, e) {
                t.Fail()
       }
}
```

Reply:
> Could use a strings.Contains instead to check that key elements of the config are present

I am in favor of doing this